src/js/components/MesosLogView.js

```diff
@@ -197,6 +204,22 @@ class MesosLogView extends mixin(StoreMixin) {
     }
   }
 
+  goToNewHighlightedSearch() {
+    let logContainer = this.getLogContainerNode();
+    let node = logContainer.querySelector('.highlight.selected');
+    if (!node) {
+      return;
+    }
+
+    let containerHeight = logContainer.clientHeight;
+    let containerScrollTop = logContainer.scrollTop;
+
+    if ((node.offsetTop > containerHeight + containerScrollTop) ||
+      node.offsetTop < containerScrollTop) {
```

Comment:
you're right, I think `offsetTop` might've had some problems. I'll create a DOMUtils method for that. I believe scrollTop was good (we use that in reactjs-components heavily for the tables)